sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -54,12 +54,12 @@ Getting Started
 ===============
 
 Bokeh is a large library that exposes many capabilities, so this section is
-only a quick tour of some common Bokeh use-cases and workflows. For more
+only a quick tour of some common Bokeh use cases and workflows. For more
```

Comment:
I did a quick Google search, and it doesn't look like this is typically hyphenated. Your style guide may state otherwise, but I wanted to flag it.